samples/SmokeTest/KeyVaultTest.ts

```diff
@@ -0,0 +1,60 @@
+import { EnvironmentCredential } from "@azure/identity";
+import { SecretsClient } from "@azure/keyvault-secrets";
+
+export class KeyVaultSecrets{
+    private static client : SecretsClient;
+    private static secretName : string;
+    private static secretValue : string;
+    
+    static async Run() {
+        console.log();
+        console.log("------------------------");
+        console.log("Key Vault - Secrets\nIdentity - Credential");
+        console.log("------------------------");
+        console.log("1) Set a secret");
+        console.log("2) Get that secret");
+        console.log("3) Delete that secret (Clean up the resource)");
+        console.log();
+        
+        // EnvironmentCredential expects the following three environment variables:
+        // * AZURE_TENANT_ID: The tenant ID in Azure Active Directory
+        // * AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
+        // * AZURE_CLIENT_SECRET: The client secret for the registered application
+        const credential = new EnvironmentCredential();
+        const url = process.env["AZURE_PROJECT_URL"] || "<YourProjectURL>";
+
+        KeyVaultSecrets.client = new SecretsClient(url,credential);
+
+        KeyVaultSecrets.secretName = "MySecretName";
+        KeyVaultSecrets.secretValue = "MySecretValue";
+
+        await KeyVaultSecrets.setSecret();
+        await KeyVaultSecrets.getSecret();
+        await KeyVaultSecrets.deleteSecret();
```

Comment:
Is it important for `deleteSecret` to cleanup the secret resource after a test? If so, then you could wrap `setSecret` and `getSecret` in a try/catch, and call `deleteSecret` in the finally block:
```typescript
try {
  await KeyVaultSecrets.setSecret();
  await KeyVayltSecrets.getSecret();
} catch (err) {
  throw err;
} finally {
  await KeyVaultSecrets.deleteSecret();
}
```

Reply:
It won't cause major problems in this case, since in the event that the secret already exists, it will only be updated. But leaving something in the resource could cause costs for the clients, so I'm going to do it in that way. Thanks!